modules/Bio/EnsEMBL/Analysis/Hive/Config/ShortncRNA.pm

```diff
@@ -0,0 +1,534 @@
+          $self->o('ncrna_dir').' blastmirna',
+      },
+      -rc_name => '6GB',
+    },
+
+    {
+      -logic_name => 'fetch_rfam_accessions',
+      -module => 'Bio::EnsEMBL::Hive::RunnableDB::SystemCmd',
+      -parameters => {
+        cmd => 'perl ' . $self->o('sncrna_analysis_script') . '/fetch_rfam_accessions.pl ' .
+          ' -h ' . $self->o('rfam_host') .
+          ' -u ' . $self->o('rfam_user') .
+          ' -p ' . $self->o('rfam_port') .
+          ' -d ' . $self->o('rfam_dbname') .
+          ' -c ' . $self->o('clade') .
+          ' -o ' . $self->o('ncrna_dir'),
+      },
+      -rc_name => 'default',
+      -flow_into => { '1' => 'extract_rfam_cm'},
+    },
+
+    {
+      -logic_name => 'extract_rfam_cm',
+      -module => 'Bio::EnsEMBL::Hive::RunnableDB::SystemCmd',
+      -parameters => {
+        cmd => 'perl ' . $self->o('sncrna_analysis_script') . '/filter_cm.pl ' .
+          $self->o('rfam_cm') . ' ' .
+          $self->o('ncrna_dir') . '/accessions.txt ' .
+          $self->o('filtered_rfam_cm'),
+      },
+      -rc_name => 'filter',
+    },
+
+    {
+      -logic_name => 'create_small_rna_slice_ids',
+      -module     => 'Bio::EnsEMBL::Analysis::Hive::RunnableDB::HiveSubmitAnalysis',
+      -parameters => {
+        coord_system_name => 'toplevel',
+        iid_type => 'slice',
+        slice_size => 1000000,
+        top_level => 1,
+        target_db => $self->o('dna_db'),
+        batch_slice_ids => 1,
+        batch_target_size => 2000000,
+      },
+      -flow_into => {
+        '2->A' => ['mirna_blast','run_cmsearch'],
+        'A->1' => ['filter_ncrnas'],
+      },
+      -rc_name    => 'default',
+    },
+
+    {
+      -logic_name => 'run_cmsearch',
+      -module     => 'Bio::EnsEMBL::Analysis::Hive::RunnableDB::HiveCMSearch',
+      -parameters => {
+        output_db => $self->o('ncrna_db'),
+        dna_db => $self->o('dna_db'),
+        logic_name => 'ncrna',
+        module     => 'HiveCMSearch',
+        cmsearch_exe_path => $self->o('cmsearch_exe_path'),
+        rfam_seeds => $self->o('rfam_seeds'),
+        rfam_cm => $self->o('filtered_rfam_cm'),
+        blast_db_dir_path => $self->o('ncrna_blast_path').'/',
+        output_dir => $self->o('ncrna_dir'),
+      },
+      -hive_capacity => 900,
+      -rc_name    => '5GB',
+      -flow_into => {
+        '-1' => 'run_cmsearch_highmem',
+        '-2' => 'run_cmsearch_highmem'},
+    },
+
+    {
+      -logic_name => 'run_cmsearch_highmem',
+      -module     => 'Bio::EnsEMBL::Analysis::Hive::RunnableDB::HiveCMSearch',
+      -parameters => {
+        output_db => $self->o('ncrna_db'),
+        dna_db => $self->o('dna_db'),
+        logic_name => 'ncrna',
+        module     => 'HiveCMSearch',
+        cmsearch_exe_path => $self->o('cmsearch_exe_path'),
+        rfam_seeds => $self->o('rfam_seeds'),
+        rfam_cm => $self->o('filtered_rfam_cm'),
+        blast_db_dir_path => $self->o('ncrna_blast_path').'/',
```

Comment:
```suggestion
        blast_db_dir_path => $self->o('ncrna_blast_path').'/',
```
If the catfiles have been replaced with catdir for ncrna_blast_path, the '/' should be removed.